.github/workflows/release.yaml

```diff
@@ -187,3 +187,84 @@ jobs:
         run: |
           ./radius/.github/scripts/release-create-tag-and-branch.sh recipes ${{ steps.get-version.outputs.release-version }} ${{ steps.get-version.outputs.release-branch-name }}
         if: success()
+
+      - name: Wait for the radius-project/radius Build and Test workflow to complete
```

Comment:
Instead of creating our own script, I would consider using the following action:

1. https://github.com/marketplace/actions/wait-on-check
2. https://github.com/marketplace/actions/await-remote-run

Reply:
wait-on-check is promising for this scenario.

Reply:
wait-on-check doesn't support different repos, so I forked and added these changes: https://github.com/lewagon/wait-on-check-action/compare/master...willdavsmith:wait-on-check-action:master